### DISCUSSION TITLE
November Release?

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3607] [General]
Started up today and notice some things have changed.  What is the "Block" button? Nothing in it when I select it.
![image](https://user-images.githubusercontent.com/94488623/144114742-2f0c6204-9241-4b5b-8f2d-c7a5b1d1bae0.png)

Also I like the new screen selection.

**Comment by twilson9034** [comment]
I have noticed that I am missing the create and update screens when a new screen was created using a non blank screen.

**Reply by joebudi** [reply]
Hey. 

When you add a blank screen you are adding a single screen with no CRUD identity attached to it. It's like this because there are times you don't need CRUD screens.

To add create and update screens, just add two new blank screens and name/update URLs accordingly.

It is probably easier to create a data table, then add the autoscreens.

**Reply by mjashanks** [reply]
@twilson9034 try clicking the blue update button in the top right:
![image](https://user-images.githubusercontent.com/3524181/144126710-dd43b6c1-eb17-4d66-8cf0-502bde61fac8.png)

This should make things appear in your "Blocks" dropdown. We should make this more obvious.

**Reply by twilson9034** [reply]
Thank you that worked for the blocks.

**Reply by twilson9034** [reply]
@joebudi I didn't select blank screen, I did select an auto generated screen. It is a duplicate so not sure if that matters or not.

**Reply by joebudi** [reply]
Ah. Apologies. I misread that.  I'll raise this with the team.

**Reply by twilson9034** [reply]
Also the ability to select an item and hit delete to delete the item is not working.

**Reply by twilson9034** [reply]
Not sure how the block are suppose to behave but when I use them nothing is appearing. using the "card" still works but not the new features.

**Reply by mjashanks** [reply]
> when I use them nothing is appearing

I can confirm this - they don't seem to work with existing apps - will look into this. They do work with new app

> the ability to select an item and hit delete to delete the item is not working

also, I see the same with existing apps, but still works with new apps.

Thanks for reporting @twilson9034.

**Reply by mjashanks** [reply]
A workaround for this - export your app, then import it again. fixes both of these issues
